### DISCUSSION TITLE
Buy paxcounter amazon but Display is reverse, how to solve?

### DISCUSSION THREAD

**Discussion by opabert** [cyberman54/ESP32-Paxcounter#839] [Q&A]
I have no idea what talking about my age 75 and like ask if possible :
- step by step instructions?
- for dummys
- how to solve display flip?
- ready made file upload?

**Comment by cyberman54** [comment]
1. Install the necessary toolchain to rebuild the software on your system: [platformio.org](https://platformio.org/)
2. Download current paxcounter software and create a platformio project: [paxcounter on github](https://github.com/cyberman54/ESP32-Paxcounter.git)
3. Identify the appropriate board configuration file for your board purchased on amazon (probably `/src/hal/ttgov21new.h`)
4. edit the file found in nr. 3 and add line `#define MY_DISPLAY_FLIP 1` to the file
5. build the firmware with platformio, as installed in nr. 1
6. upload the firmware to the board with platformio

**Comment by opabert** [comment]
like mention my age 75 years this is new for me
Is there a ready file to upload to LilyGo paxcounter board?


![PAXCOUNTER](https://user-images.githubusercontent.com/36929496/147983752-1587579a-5cfb-41c1-8c48-9cabd2922641.jpg)
?

**Reply by cyberman54** [reply]
Please post amazon link for what you've bought.

**Reply by opabert** [reply]
https://www.amazon.nl/gp/product/B07SFJ86FG/ref=ppx_yo_dt_b_asin_title_o09_s01

**Comment by opabert** [comment]
Like ask anyone able to help me out?

**Reply by t-huyeng** [reply]
What exactly are your problems, where do you need help?
